packages/@aws-cdk-testing/framework-integ/test/aws-kms/test/integ.key-multi-region.ts

```diff
@@ -0,0 +1,19 @@
+import { App, Stack } from 'aws-cdk-lib';
+import * as kms from 'aws-cdk-lib/aws-kms';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+
+class TestStack extends Stack {
+  constructor(scope: App) {
+    super(scope, 'TestStack');
```

Comment:
How about making the stack name specific to this integ test? Because there are often other integ tests with stacks of the same name, which will fail if we try to run them in parallel.

Reply:
I wasn't considering about the conflict with other stacks. Thanks for telling me.